Shopify/shopify-cli
# Empty extension.config.yml

### WHY are these changes introduced?

Product subscription extensions currently don't work with the new development server because they don't have an `extension.config.yml`.


### WHAT is this pull request doing?

- Enables extension projects to not not have an extension.config.yml at all or an empty one.
- Fixes development entry fallback.


### How to test your changes?

```
$ shopify extension create # of type product subscription
$ cd <extension-project-name>
$ shopify extension build
```

Verify that build/main.js has been produced and contains JavaScript

### Post-release steps

This is still behind a beta flag. No post release steps required.

### Update checklist
